source/steamshim/CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
 project(wf_steam)
 
 add_definitions(
-    -DGAME_CLIENT_LAUNCH_NAME="${QFUSION_CLIENT_NAME}${CMAKE_EXECUTABLE_SUFFIX}"
-    -DGAME_SERVER_LAUNCH_NAME="${QFUSION_SERVER_NAME}${CMAKE_EXECUTABLE_SUFFIX}"
-    -DGAME_TVSERVER_LAUNCH_NAME="${QFUSION_TVSERVER_NAME}${CMAKE_EXECUTABLE_SUFFIX}"
+    -DSTEAM_BLOB_LAUNCH_NAME="wf_steam${CMAKE_EXECUTABLE_SUFFIX}"
```

Comment:
Now that you reversed this, you probably want to swap the dependencies around in `source/client/CMakeLists.txt`, so that `add_dependencies(wf_steam ${QFUSION_CLIENT_NAME})` becomes `add_dependencies(${QFUSION_CLIENT_NAME} wf_steam)`.
This ensures that `wf_steam` is built when `${QFUSION_CLIENT_NAME}` (the game) is being built.